experimental/templates/multiplatform-template/README.md

```diff
@@ -1,70 +1,100 @@
 or use Gradle:
 
 `./gradlew installDebug` - install Android application on an Android device (on a real device or on an emulator)
 
 ## iOS
 
+Make sure you have set up your environment for mobile development with Kotlin Multiplatform. A detailed guide on how to set up your environment is available in the [Kotlin Multiplatform documentation](https://kotlinlang.org/docs/multiplatform-mobile-setup.html).
+
 To work with the iOS target you need:
-- [Xcode](https://developer.apple.com/xcode/) (to setup the environement)
+- A machine running a recent version of macOS
+- [Xcode](https://developer.apple.com/xcode/) (to setup the environment)
 - [Android Studio](https://developer.android.com/studio)
-- ["Kotlin Multiplatform Mobile" plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile) (to work with all supported targets, including iOS)
+- [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile) (to work with all supported targets, including iOS)
 - [CocoaPods](https://kotlinlang.org/docs/native-cocoapods.html)
 
-Before trying to open the project in Android Studio, make sure that you have a correct environment for iOS development. 
-Try to follow official Apple tutorials how to create a sample project in Xcode 
-and register a simulator or a real device there.
+**Before opening the project in Android Studio**, make sure that your environment is set up for iOS and Kotlin Multiplatform development.
+Use [`kdoctor`](https://github.com/Kotlin/kdoctor) to ensure your development environment is configured correctly.
+
+We suggest going through the "Hello, World" steps of creating and deploying a sample project in Xcode to a simulator and/or your physical device.
+A video tutorial for setting up Xcode and running your first "Hello, World" application is available in [this Standford CS193P lecture recording](https://youtu.be/bqu6BquVi2M?start=716&end=1399).
 
 ### Running on an iOS simulator
 
-If you have a correct environment, Android Studio will show available iOS simulators on which you can run the application
-in `iosApp` run configuration.
+Once you have configured your environment correctly, you will be able to select which iOS simulator to run your application in Android Studio on by modifying the `iosApp` run configuration.
+
+Select "Run" | "Edit Configurations..." and navigate to the "iOS Application" | "iosApp" run configuration. In the "Execution target" drop-down, select your target device.
 
 ### Running on a real iOS device
 
-To run on a real iOS device you need:
-- [Team ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect.)
-- set this Team ID in `iosApp/Configuration/Config.xcconfig` in the `TEAM_ID` field
+Running your Compose Multiplatform application on a physical device can be done for free. You need:
+- an [Apple ID](https://support.apple.com/en-us/HT204316)
```

Comment:
I think in a link about free - means creating Apple ID, but not a launch on a physical device.

From my point of view, it looks that to launch on a real device, developers need to have a paid Developer account, or be a member of Team with paid account admin.

Reply:
But, maybe I am wrong.
I will try your instruction `Running with a free Personal Team`.

Reply:
It should work just fine with a free Apple ID, since that's what I'm doing with my iPhone 😆

Reply:
I checked, it works!
Thanks for that detailed instruction!